common-custom-user-data-scripts/gradle-common-custom-user-data.gradle

```diff
@@ -341,7 +341,7 @@ String appendIfMissing(String str, String suffix) {
     str.endsWith(suffix) ? str : str + suffix
 }
 
-String trimAtEnd(String str) {
+static String trimAtEnd(String str) {
     ('x' + str).trim().substring(1)
 }
 
```

Comment:
Please verify the script still works with 5.0.x, 5.6.x and 6.0.x. (not with regard to configuration caching but the general functionality)

Reply:
Gradle 5.6.4
https://scans.gradle.com/s/gba6luiyltwp2

Gradle 5.0
https://scans.gradle.com/s/vwtpepkn6ooiq

Gradle 6.6.1 using configuration cache
https://scans.gradle.com/s/xzqryy5s6tg4s

There were other problems with the configuration cache in 6.6.1 with this build, but the git status, git commit and git link were correctly added.